R/dist_spec.R

```diff
@@ -698,11 +699,12 @@ plot.dist_spec <- function(x, samples = 50L, res = 1, cumulative = TRUE, ...) {
 
   # Plot PMF and CDF as facets in the same plot
   plot <- ggplot(
-    pmf_data, mapping = aes(x = x, y = p, group = sample, linetype = type)
+    pmf_data, mapping = aes(x = x, y = p, group = sample, color = type)
   ) +
     geom_line() +
     facet_wrap(vars(distribution)) +
     labs(x = "x", y = "Probability") +
+    scale_color_brewer(palette = "Set1") +
```

Comment:
```suggestion
    scale_color_brewer(palette = "Dark2") +
```
it's what we're using elsewhere.

Reply:
I didn't want to confuse the palette for the estimation functions with this but happy to use the same palette.